examples/layers/preprocessing/segmentation/mix_up_demo.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2023 The KerasCV Authors
```

Comment:
This is great! Could you please add a screenshot of the result of running this script to the PR description?

Reply:
Update. Thank you. @ianstenbit